src/features/auth/Register.tsx

```diff
@@ -105,9 +106,9 @@ const RegisterPage = () => {
     >
       <Card sx={{ width: 400, p: 3, boxShadow: 3 }}>
         <CardContent>
-          <Typography variant='h6' color='primary' align='center'>
-            Logo
-          </Typography>
+          <Box sx={{ display: 'flex', justifyContent: 'center', alignItems: 'center', m: 1 }}>
+            <Logo style={{ height: 40, width: 40, marginRight: 10 }} fill='#28a2a2' />
```

Comment:
note: I tried to refactor this into a reusable Logo component, however the svg would not pass the color or fill in. So I had to do it this way